datashuttle/datashuttle.py

```diff
@@ -389,7 +400,71 @@ def get_appdir_path(self):
         return os.fspath(appdir_path)
 
     def get_remote_path(self):
-        return self.cfg["remote_path"].as_posix()
+        """
+        Force remote path to return as posix as if local fs is windows and remote is
```

Comment:
I guess fs is filesystem? I'd expand these acronyms so it's easier to parse in a years time.